--- ext2ed-0.1/ext2ed.h~	2004-04-17 13:17:10.033449188 +0200
+++ ext2ed-0.1/ext2ed.h	2004-04-17 13:17:39.635893456 +0200
@@ -34,7 +34,7 @@ Copyright (C) 1995 Gadi Oxman
 
 #define DEBUG						/* Activate self-sanity checks */
 
-#include <linux/ext2_fs.h>				/* Main kernel ext2 include file */
+#include <ext2fs/ext2_fs.h>				/* Main kernel ext2 include file */
 #include <sys/stat.h>
 
 #ifdef OLD_NCURSES					/* The ncurses interface */
